app/jobs/check_catalogues_job.rb

```diff
@@ -0,0 +1,89 @@
+class CheckCataloguesJob < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+  require_relative '../../app/jobs/scrape_catalogues_job.rb'
+  require_relative '../../app/jobs/delete_catalogues_job.rb'
+  require_relative '../../app/jobs/capybara_quit_job.rb'
+
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist
+
+  def perform
+
+    #new_catalogues stores result from calling catalogue_gather method
+    new_catalogues = catalogue_gather
+
+    #collect current_catalogue data
+    current_catalogue = Hash.new(0)
+    Catalogue.all.each{|c| current_catalogue[c.id] = c.catalogue_num}
+
+    #scrape new posts from new catalogue
+    catalogue_to_add = add_new_catalogue(current_catalogue, new_catalogues)
+    ScrapeCataloguesJob.perform_later(catalogue_to_add)
+
+    #delete old and duplicate catalogues
+    delete_old_duplicate_catalogue(current_catalogue, new_catalogues, catalogue_to_add)  
+
+    #Capybara quit session
+    CapybaraQuitJob.perform_later
+  end
+
+  private
+
+  def catalogue_gather
+    regions = ["Melbourne, 3000", "Sydney, 2000", "Brisbane city, 4000", 'Perth, 6000']
+    catalogue_result = []

```

Comment:
If I'm understanding correctly, `published_catalogue_nums` would be a more descriptive variable name for `catalogue_result`.

Consider renaming `catalogue_gather` to `scrape_published_catalogue_nums`